util/dasharo/tag-with-changelog

```diff
@@ -396,14 +402,18 @@ for module in $(git submodule --quiet foreach pwd); do
 	get_log_submodule "${!old_version}" "${!new_version}" "${path}"
 done
 
-# Add the collected data to the top of the existing logfile for parsing
-if [ -n "$UPDATE_MAIN_LOGFILE" ]; then
-	tmpfile="$(mktemp "LOGFILE.XXXX")"
-	grep -Fxv -f "$MAIN_LOGFILE" "$LOGFILE" > "$tmpfile"
-	printf "\n\n" >> "$tmpfile"
-	cat "$MAIN_LOGFILE" >> "$tmpfile"
-	mv "$tmpfile" "$MAIN_LOGFILE"
-	rm -f "$LOGFILE"
+# Create signed tag, replace existing, use LOGFILE as message, open editor
+git tag -s -f -F $LOGFILE -e $NEW_GIT_VERSION
```

Comment:
I'd add quotes around `$LOGFILE` here and below. Version is unlikely to need escaping, but a full path could.  Although, I'd add quotes around `$NEW_GIT_VERSION` too, just because if it's empty or weird in some way, the errors will be more intelligible.